client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -162,7 +123,7 @@ export const failedJITM = ( action ) => ( dispatch, getState ) => {
 registerHandlers( 'state/data-layer/wpcom/sites/jitm/index.js', {
 	[ JITM_FETCH ]: [
 		dispatchRequest( {
-			fetch: isRunningInJetpackSite ? doJetpackFetchJITM : doFetchJITM,
+			fetch: doFetchJITM,
```

Comment:
I think you still need to keep the non-Jetpack and Jetpack versions of the fetch because one of them has the `/sites/:s` prefix and the other doesn't.

Reply:
Agreed. Modified the path.